### PR TITLE
add azuredisk-issues.md & azurefile-issues.md

### PR DESCRIPTION
**Pull request by andyzhangx**
@karataliu @feiskyer

### CODE REVIEW

File: docs/azuredisk-issues.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
+# azure disk plugin known issues
+### Recommended stable version for azure disk
+| k8s version | stable version |
+| ---- | ---- |
+| v1.7 | 1.7.14 |
+| v1.8 | 1.8.13 |
+| v1.9 | 1.9.7 (1.9.6 on AKS) |
+| v1.10 | 1.10.2 |
+
```
Comment (feiskyer) [inline docs/azuredisk-issues.md]: Could we add a TOC for both files?

File: docs/azuredisk-issues.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
+# azure disk plugin known issues
+### Recommended stable version for azure disk
+| k8s version | stable version |
+| ---- | ---- |
+| v1.7 | 1.7.14 |
+| v1.8 | 1.8.13 |
+| v1.9 | 1.9.7 (1.9.6 on AKS) |
+| v1.10 | 1.10.2 |
```
Comment (feiskyer) [inline docs/azuredisk-issues.md]: I think those stable version means above it? may use v1.10.2+?

File: docs/azuredisk-issues.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+
+### 1. disk attach error
```
Comment (feiskyer) [inline docs/azuredisk-issues.md]: nit: use `##` for those titles

File: docs/azuredisk-issues.md [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+**Issue details**:
```
Comment (feiskyer) [inline docs/azuredisk-issues.md]: nit: use `###` converting it to third level titles

**Comment by andyzhangx** [comment]
@feiskyer can we merge this first?

**Comment by feiskyer** [comment]
@andyzhangx ok, let's get this in first and format it in a following PR.

/lgtm
/approve

**Pull request merged**